context/opencensus-context/examples/thread_pool.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from multiprocessing.dummy import Pool as ThreadPool
```

Comment:
Why `multiprocessing...ThreadPool` instead of `concurrent.futures.ThreadPoolExecutor`?

Reply:
This former works with Python 2.7.